### DISCUSSION TITLE
已更新v3.12.2，但是存储选项里没有阿里云盘Open选项

### DISCUSSION THREAD

**Discussion by Zliy997** [AlistGo/alist#3686] [Q&A] [answered]
<img width="297" alt="image" src="https://user-images.githubusercontent.com/67785977/222509297-b26b459b-ce58-4e0f-9e64-8bad324bddac.png">

**Comment by LUCASHAN76** [comment, marked as answer]
重启alist

**Comment by bg4evk** [comment]
重启
